src/component/common/ConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/ConstraintFormHeader/ConstraintFormHeader.tsx

```diff
@@ -14,11 +15,7 @@ export const ConstraintFormHeader: React.FC<
 > = ({ children, ...rest }) => {
     const styles = useStyles();
     return (
-        <h3
-            className={styles.header}
-            {...rest}
-            style={{ marginBottom: '0.25rem' }}
-        >
+        <h3 className={styles.header} {...rest}>
```

Comment:
Would put `{...rest}` before the `className` prop to make sure `styles.header` doesn't get overridden.